config/sample.env

```
@@ -8,11 +8,17 @@ QUERIDO_DIARIO_SUGGESTION_SENDER_EMAIL=example@email.com
 QUERIDO_DIARIO_SUGGESTION_RECIPIENT_NAME=Recipient Name
 QUERIDO_DIARIO_SUGGESTION_RECIPIENT_EMAIL=example@email.com
 QUERIDO_DIARIO_SUGGESTION_MAILJET_CUSTOM_ID=AppCustomID
-POSTGRES_USER=companies
-POSTGRES_PASSWORD=companies
-POSTGRES_DB=companiesdb
-POSTGRES_HOST=localhost
-POSTGRES_PORT=5432
+QUERIDO_DIARIO_FILES_ENDPOINT=s3://queridodiariobucket/
```

Agora que percebi a variável de ambiente, é melhor deixar um valor que já funcionaria com o setup local
```suggestion
QUERIDO_DIARIO_FILES_ENDPOINT=http://localhost:9000/queridodiariobucket/
```

---

ok, irei mudar agora mesmo